From: Kylie McClain <somasis exherbo org>

FD_ZERO, fd_set, etc are defined within sys/select.h on musl.

Signed-off-by: Kylie McClain <somasis exherbo org>
---
 utils/ndptool.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/ndptool.c b/utils/ndptool.c
index 1d96f4c..96479fa 100644
--- a/utils/ndptool.c
+++ b/utils/ndptool.c
@@ -28,6 +28,7 @@
 #include <arpa/inet.h>
 #include <errno.h>
 #include <ndp.h>
+#include <sys/select.h>
 
 enum verbosity_level {
 	VERB1,
-- 
2.8.2

